src/app/utils/task.js

```diff
@@ -0,0 +1,19 @@
+export default class Task {
+  constructor(index = null, desc = '', completed = false) {
+    this.index = index;
+    this.desc = desc;
+    this.completed = completed;
+  }
+
+  complete = () => {
+    if (this.completed === false) {
+      this.completed = true;
+    } else {
+      this.completed = false;
+    }
+  }
+
+  edit = (desc) => {
+    this.desc = desc;
+  }
+}
```

Comment:
After creating each individual task, you could destructure the objects inside the array to help with maintainability. It makes changing an object easier.